recipes/idiap-devtools/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "idiap-devtools" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/idiap-devtools-{{ version }}.tar.gz
+  sha256: 97d8b0b1b37ff808d0c24c27069bb64aa16eed2f30e91e2cb18d7d88a2252012
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=61.0.0
+    - wheel
+    - pip
+  run:
+    - python >=3.9
+    - click >=8
+    - cookiecutter
+    - gitpython
+    - packaging
+    - python-dateutil
+    - python-gitlab
+    - pytz
+    - pyyaml
+    - setuptools
+    - tomli
+    - tomlkit
+    - xdg
+    - __unix
```

Comment:
I'm not sure this is needed (the skip build above should fix that):
```
  - __unix
```

Reply:
Thanks for your review.  Please note this is a `noarch: python` package for unix-only (macOS or Linux).  However, as conda-forge is setup, noarch packages are still installable on Windows environments.  This runtime requirement guarantees this package won't be installable on Windows.  It was recommended to me by @beckermr.